builder/Dockerfile.windows

```diff
@@ -1,10 +1,12 @@
+# Base image depends on target platform.
+# See: https://mcr.microsoft.com/en-us/product/windows/nanoserver/about
+ARG BASE_IMAGE=mcr.microsoft.com/windows/nanoserver:ltsc2019
+
 # Builder
-FROM golang:1.18.0-windowsservercore-1809 AS builder
+# golang builder image is multi-platform
+FROM golang:1.19.5 AS builder
 
-ENV GO111MODULE=on \
-    CGO_ENABLED=0 \
-    GOOS=windows \
-    GOARCH=amd64
```

Comment:
just highlighting the diff, is GOARCH no long necessary? assume there is no support of windows 2019 on arm.

Reply:
These are already set in the whichever base builder image is referenced in the Dockerfile(s), which should be appropriate for the platform on which the build happens, e.g.:
```bash
docker run --rm golang:1.9.5 go env
GOARCH="amd64"
...
GOOS="linux"
```